src/modules/content-sqlite/content-sqlite.c

```diff
@@ -213,7 +226,9 @@ static int content_sqlite_store (struct content_sqlite *ctx,
     uint8_t hash[BLOBREF_MAX_DIGEST_SIZE];
     int hash_len;
     int uncompressed_size = -1;
+    struct timespec t0;
 
+    monotime (&t0);
```

Comment:
I could be reading the diff wrong, but it looks like `t0` isn't used elsewhere in this function? Apologies if I just missed it.

Reply:
Oh good catch!